src/testdir/test_blockedit.vim

```diff
@@ -28,4 +28,12 @@ func Test_blockinsert_delete()
   bwipe!
 endfunc
 
+func Test_blockappend_eol_cursor()
```

Comment:
There is a commented out test for this in the Test_visual_block_mode() function
in test_visual.vim (search for BUG).

Reply:
ah, missed that part of the message.

Reply:
BTW: I remember always having wondered, why there is the condition for `bd.is_MAX`. I hope this does not cause any problem.s